draft-ietf-quic-http.md

```diff
@@ -370,6 +370,10 @@ able to send stream data first after the cryptographic handshake completes.
 
 ### Push Streams
 
+Server push is an optional feature introduced in HTTP/2 that allows a server to
+initiate a response before a request has been made.  See section 8.2 of
+{{!RFC7540}} for more details.
```

Comment:
That's not quite accurate... since the response has to be associated with a previous request. I don't think that level of detail is necessary though. How about "The Server Push feature of HTTP/2 (see Section 8.2 of {{!RFC7540}}) is facilitated in HTTP/3 via push streams."

Reply:
I'm not too fond of relying on HTTP/2 to understand HTTP/3 beyond - "see this list header and verbs".

Reply:
Understandably -- it's a line we have to walk.  (Note that HTTP/2 defines no verbs and pseudo-no headers.)

My principle has been that I'm trying to weed out assumptions that you've read RFC7540 and understand a given concept already.  I'm happy to save redundant and possibly-conflicting text by pointing to a specific section of RFC7540, however.